relayer/txmgr/metrics.go

```diff
@@ -13,4 +13,25 @@ var (
 		Name:      "resend_total",
 		Help:      "The total number of transaction resends to a destination chain",
 	}, []string{"chain"})
+
+	txL1GasFee = promauto.NewGaugeVec(prometheus.GaugeOpts{
+		Namespace: "relayer",
+		Subsystem: "txmgr",
+		Name:      "tx_fee_gwei",
+		Help:      "L1 gas fee for transactions in GWEI",
```

Comment:
I think this should actually be a histogram, since individual submissions's fee are uncorrelated. And I would rather just track `GasUsed`.